### DISCUSSION TITLE
Problemi con la guida Haaska-Aws per Home Assistant

### DISCUSSION THREAD

**Discussion by Faermas** [indomus/forum#3255] [GENERALE - Discussioni generiche]
Buonasera a tutti, ho un problema con l'integrazione di Alexa mediante la procedura Haaska-AWS. Ho seguito la vostra guida (l'ho rifatto per tre volte) è i vari test mi danno sempre esito positivo quindi credo di aver fatto tutto giusto ma quando vado su Alexa a cercare la mia skill non la trovo, come mai? Ho visto anche che dalla guida alla procedura sono cambiate diverse cose (tipo le "schede" da dove inserire le varie stringhe e il python) ma nonostante ciò ho messo i vari set che erano sulla guida. Non allegato codici o altro in quanto non mi compare nessun errore.

**Comment by KC-inDomus** [comment]
ribattila tutta, hai commesso qualche errore.
